src/CommonAssemblyInfo.cs

```diff
@@ -3,9 +3,9 @@
 using System.Resources;
 using System.Runtime.InteropServices;
 
-[assembly: AssemblyVersion("2.1.0")]
-[assembly: AssemblyFileVersion("2.1.0")]
-[assembly: AssemblyInformationalVersion("2.1.0")]
+[assembly: AssemblyVersion("2.0.1")]
+[assembly: AssemblyFileVersion("2.0.1")]
+[assembly: AssemblyInformationalVersion("2.0.1")]

```

Comment:
Yep, I'll back out this change now that #736 is merged.

TBH I don't think I've ever used `set_version[new_version]` :grin: 

The 2.0.1 release was here #733 - now withdrawn.